tests/shared/core/training_data/story_reader/test_common_story_reader.py

```diff
@@ -51,18 +47,9 @@ async def test_can_read_test_story(stories_file: Text, default_domain: Domain):
     assert tracker.events[4] == ActionExecuted("action_listen")
 
 
-@pytest.mark.parametrize(
-    "stories_file",
-    [
-        "data/test_stories/stories_checkpoint_after_or.md",
```

Comment:
Is it ok to stop testing this with markdown?

Reply:
good catch, returned it back